src/platform/graphics/render-pass.js

```diff
@@ -93,7 +93,7 @@ class RenderPass {
     /** @type {string} */
     name;
 
-    /** @type {import('../graphics/render-target.js').RenderTarget} */
+    /** @type {import('../graphics/render-target.js').RenderTarget|null} */
```

Comment:
this can be `undefined` as well, which means 'not rendering to any render target'. `null` means 'rendering the backbuffer'.

Reply:
Thank you! I added `undefined` aswell in next commit